plugins/devtools/src/components/Content/InfoContent/InfoContent.tsx

```diff
@@ -54,29 +55,40 @@ const useStyles = makeStyles((theme: Theme) =>
   }),
 );
 
-const copyToClipboard = ({ about }: { about: DevToolsInfo | undefined }) => {
-  if (about) {
-    let formatted = `OS: ${about.operatingSystem}\nResources: ${about.resourceUtilization}\nnode: ${about.nodeJsVersion}\nbackstage: ${about.backstageVersion}\nDependencies:\n`;
-    const deps = about.dependencies;
-    for (const key in deps) {
-      if (Object.prototype.hasOwnProperty.call(deps, key)) {
-        formatted = `${formatted}    ${deps[key].name}: ${deps[key].versions}\n`;
-      }
-    }
-    window.navigator.clipboard.writeText(formatted);
-  }
-};
-
 /** @public */
 export const InfoContent = () => {
   const classes = useStyles();
+  const alertApi = useApi(alertApiRef);
+  const [infoCopied, setInfoCopied] = useState(false);
   const { about, loading, error } = useInfo();
+  const [state, setCopied] = useCopyToClipboard();
+
+  const copyToClipboard = () => {
+    if (about) {
+      let formatted = `OS: ${about.operatingSystem}\nResources: ${about.resourceUtilization}\nnode: ${about.nodeJsVersion}\nbackstage: ${about.backstageVersion}\nDependencies:\n`;
+      const deps = about.dependencies;
+      for (const key in deps) {
+        if (Object.prototype.hasOwnProperty.call(deps, key)) {
+          formatted = `${formatted}    ${deps[key].name}: ${deps[key].versions}\n`;
+        }
+      }
+      setCopied(formatted);
+      setInfoCopied(true);
+    }
+    if (state.error) {
```

Comment:
Could you add a test for this? I don't think this code will actually trigger an alert

Reply:
you mean need to right test case? or modify the code

Reply:
Yep it'd be best to add a test case for this to make sure it works